linkbrary/src/functions/useGetJson.js

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect, useCallback } from "react";
+
+const useGetJson = (func, param = null) => {
+  const [values, setValues] = useState([]);
+  const HandleLoad = useCallback(async () => {
```

Comment:
```suggestion
  const handleLoad = useCallback(async () => {
```
함수는 Hungurian Notation으로 일반적으로 써줘요